README.md

```diff
@@ -23,39 +23,129 @@ Also refer to our
 [Ackama README Template](https://github.com/ackama/wiki/wiki/Ackama-README-Template)
 wiki page to better customise this README
 
+## Purpose
+
+_Fill in purpose of this project_
+
 ### Project Setup
 
-#### Clone and Clean
+#### Create your project from this repository
 
-Clone this repository, remove its `.git` directory and initialise it as a new
-repository:
+* Remove this title after setting up the project, to avoid confusion *
 
+Execute the following to create a new project based of this repository, updated
+to the latest version of the SilverStripe stack
+
+```bash
+$ composer create-project --no-install --remove-vcs ackama/silverstripe-cwp-template your-cwp-project
```

Comment:
I forgot that you do this through composer, meaning we could easily have a custom script to generate files if we wanted - but probably best to land that in a followup PR as an improvement; this ones already pretty big 😅

Reply:
I didn't get this comment 🤔

Reply:
@G-Rath could you rephrase please? :)

Reply:
This was in relation to [this comment](https://github.com/ackama/silverstripe-cwp-template/pull/20#discussion_r602944302).

Feel free to just ignore it :)

Reply:
@G-Rath gotcha. Yeah, will ignore until we come up with a better idea